test/e2e/bootstrapper_test.go

```diff
@@ -62,7 +62,7 @@ func TestBootstrapper(t *testing.T) {
 	t.Run("Uninstall kubelet without kubelet service present", testUninstallWithoutKubeletSvc)
 
 	// Run the bootstrapper, which will start the kubelet service
-	wmcb, err := bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath, "", "", "", "")
+	wmcb, err := bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath, "", "", "", "", "aws")
```

Comment:
Why not use the constant `configv1.AWSPlatformType` for this instead of a raw string?

Reply:
we need to add openshift/api in the dependencies then...

Reply:
with all its dependencies it is 110k lines of code (146 files changed, 109510 insertions(+), 553 deletions(-)) just for a string constant.

Reply:
that's why I prefer not to do this